kitsune/customercare/forms.py

```diff
@@ -0,0 +1,28 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _lazy
+
+from kitsune.customercare.zendesk import ZendeskClient, CATEGORY_CHOICES, OS_CHOICES
+
+
+class ZendeskForm(forms.Form):
+    """Form for submitting a ticket to Zendesk."""
+
+    product = forms.CharField(widget=forms.HiddenInput)
```

Comment:
Product is not validated at all. I managed to change the field in the form and successfully post to Zendesk.